### PR TITLE
Update health check script to use any hostname

### PR DESCRIPTION
**Pull request by nalyanyam**
This allows HEALTHCHECK routine to work with any container hostname.

### CODE REVIEW

File: docker/puppetserver-standalone/Dockerfile [hunk 1]
```diff
@@ -51,12 +51,13 @@ ENTRYPOINT ["dumb-init", "/docker-entrypoint.sh"]
 CMD ["foreground"]
 
 HEALTHCHECK --interval=10s --timeout=10s --retries=90 CMD \
+  hostname=$(puppet config print certname)
```
Comment (underscorgan) [inline docker/puppetserver-standalone/Dockerfile]:
```suggestion
  hostname=$(puppet config print certname) && \
```

**Comment by nalyanyam** [comment]
Updated.

**Comment by nalyanyam** [comment]
Thanks @underscorgan

**Comment by nalyanyam** [comment]
@underscorgan , was just checking healthcheck.sh script again, and it looks like we should have put double quotes. the script still works but it does not interpolate the hostname in resolve line.
-   --resolve '${hostname}:8140:127.0.0.1' \
+  --resolve "${hostname}:8140:127.0.0.1" \

Should i send another PR to fix it?

**Comment by underscorgan** [comment]
@nalyanyam ah, yes, good catch. PR would be great.

**Pull request merged**